sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample1_DetectEntireSeriesAnomaly.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Azure;
```

Comment:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```
Above text is missing at the top of the file.

Reply:
Removed this file.

---
In reply to: [481469355](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r481469355) [](ancestors = 481469355)